### DISCUSSION TITLE
Daily 4/11

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#274] [General]
Ayer estuve en las clases de SGC y full stack. También empecé con la parte del diagrama de clases de controladores y servicios. Hoy voy a seguir con ese diagrama y estar en la clase de full-stack. También quiero empezar con mi portfolio personal en mi Wordpress local, y empezar con la documentación final sobre el proyecto.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve practicando para el video, hice algunos intentos en OBS, es no es mucho lo que se puede decir en un poco mas de un minuto y medio. Recomiendo hacer varias pruebas antes de la grabaciòn del domingo. También estuve leyendo los contenidos nuevos de SGC. Luego participé de las clases sincrónicas de SGC y Full Stack. 
Hoy tengo clases de Inglés y luego luego tengo examen de otro curso. Voy a terminar de darle forma a mi portfolio personal para ya subir las capturas a las tareas de SGC. También vi que hay que agregar algunos detalles a los formularios de la página, que lo estaré realizando el fin de semana.

**Comment by AhumadaCeleste** [comment]
Ayer estuve con python. Hoy viendo clases de SGC y Full Stack y estudiando material de ambos espacios curriculares
